docs/src/input.md

```diff
@@ -3,8 +3,7 @@ id: input
 title: "Input"
```

Comment:
Some ideas: "Interacting with the page", "Performing actions", "Input elements".